ansible/roles/ansible-horizon-plugin/tasks/find_manage_file.yml

```diff
@@ -1,11 +1,17 @@
 ---
 
-- name: find location of manage.py
-  shell: "find {{ ENV_PATH }}  -name '*manage.py' | grep -E 'openstack-dashboard|horizon|bin'"
+- name: Find location of manage.py
+  find:
+    paths: "{{ ENV_PATH }}"
+    file_type: file
+    use_regex: yes
+    patterns: "^.*manage.py$"
```

Comment:
Have we tested this?

Reply:
Yes, I have tested the horizon plugin service installations and un-installation on the OSA Wallaby CentOS8 Stream platform.
Deployment is passed without any issue.